packages/terra-theme-context/src/ThemeContext.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const ThemeProviderContext = React.createContext({});
+
+const contextShape = PropTypes.shape({
+  /**
+   * The current application theme className.
+   * The default theme is indicated as undefined.
+   */
+  className: PropTypes.string,
```

Comment:
`className` seems a little weird to me. I would have expected this to be `name`, `theme`, or `themeName`.  But if its intention is targeted towards applying a className I think it makes sense.

Reply:
it is targeted towards applying a class name, I thought about adding a 'name' prop as well to give a named to the theme? That way we could be very explicit about default but it also feels possibly redundant :
```js
const theme = {
  name: 'default',
  className: undefined,
};

const theme = {
  name: 'clinical-lowlight-theme',
  className: 'clinical-lowlight-theme',
};
```